src/ed25519.rs

```diff
@@ -0,0 +1,117 @@
+use crate::Commit;
+use bigint::U256;
+use curve25519_dalek::edwards::{CompressedEdwardsY, EdwardsPoint};
+
+pub use curve25519_dalek::{constants::ED25519_BASEPOINT_POINT as H, scalar::Scalar};
+
+pub type Point = EdwardsPoint;
+
+lazy_static::lazy_static! {
+    /// Alternate generator of ed25519.
```

Comment:
From where? I guess Monero uses one. Is this it?

Reply:
Sorry, this should have been documented. It's the hash of `ED25519_BASE_POINT`. This is originally used in Monero Ring Confidential Transactions https://github.com/monero-project/monero/blob/9414194b1e47730843e4dbbd4214bf72d3540cf9/src/ringct/rctTypes.h#L454